spec/api-menu-spec.js

```diff
@@ -860,11 +860,15 @@ describe('Menu module', () => {
           submenu: [
             {
               label: 'Test Item',
-              accelerator: 'Ctrl+T',
-              click: () => {
-                // Test will succeed, only when the menu accelerator action
-                // is triggered
-                Promise.resolve()
```

Comment:
Could we also have a test for the inverse, where `triggeredByAccelerator` _is_ true?

Reply:
We are testing to see if it's true on [line 870](https://github.com/electron/electron/pull/18865/files/a7f946ea8bf681752e36763388178e15bf87d245#diff-0f899d7673156733456b3d561fe6c8feR870) here. Do you mean testing a manual click to see if `triggeredByAccelerator` will show up as false?

Reply:
My comment was intended to be in line with https://github.com/electron/electron/pull/18865/files#r295039742, but i think i wasn't super clear: i think that we should be checking to ensure that if we change line 870 to `triggeredByAccelerator: false`, this test should fail.